src/porepy/compositional/compositional_mixins.py

```diff
@@ -1022,66 +1021,64 @@ def fraction(domains: pp.SubdomainsOrBoundaries) -> pp.ad.Operator:
         return fraction
 
 
-class FluidMixtureMixin:
-    """Mixin class for modelling a mixture and providing it as an attribute to a PorePy
-    model.
+class FluidMixin(PorePyModel):
+    """Mixin class for introducing a general fluid (mixture) into a PorePy model and
+    providing it as an attribute :attr:`fluid`.
+
+    Fluid properties are by definition expressed through respective phase properties,
+    which can be overriden here as part of the constitutive modelling.
 
-    Provides means to create domain-dependent callables representing thermodynamic
-    properties which appear in equations.
+    The following methods provide expressions for them:
```

Comment:
What does "them" refer to?

Reply:
Phase properties. Will make this more clear.

Reply:
Done.